s3/src/main/scala/akka/stream/alpakka/s3/model.scala

```diff
@@ -104,13 +114,13 @@ object MultipartUploadResult {
       location: akka.http.javadsl.model.Uri,
       bucket: String,
       key: String,
-      etag: String,
+      eTag: String,
```

Comment:
AFAIK this can break source compatibility (i.e. if someone calls `create` with named parameters) but it doesn't break binary compatibility.